### PR TITLE
Update Leaflet to v1.0.3

### PR DESCRIPTION
**Pull request by fbonzon**
Update other JavaScript dependencies.

Fix lint to avoid:
src/main/webapp/js/graphhopper/GHRequest.js: line 272, col 27, ['key'] is better written in dot notation.
src/main/webapp/js/graphhopper/GHRequest.js: line 273, col 2, Missing semicolon.

I did test and verified nothing broke. Please do some testing too. Demo at: http://françois.ch/gh/

### CODE REVIEW

File: web/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,11 @@
...
         <meta name="description" content="A fast route planner for biking, hiking and more! Based on OpenStreetMap including elevation data. Try out for free!"/>
-        <link rel="stylesheet" href="css/leaflet.css?v=1.0.2" />
+        <link rel="stylesheet" href="css/leaflet.css?v=1.0.3" />
         <link rel="stylesheet" href="css/leaflet.elevation-0.0.4.css" />
         <link rel="stylesheet" href="css/leaflet_numbered_markers.css" />
         <link rel="stylesheet" href="css/leaflet.contextmenu.css" />
-        <link rel="stylesheet" href="css/leaflet.loading.css" />
+        <link rel="stylesheet" href="css/leaflet.loading.css?v=0.1.24" />
         <link rel="stylesheet" href="css/ui-lightness/jquery-ui.min.css" />
         <script type="text/javascript" src="js/main.js?v=0.9.0"></script>
```
Comment (karussell) [inline web/src/main/webapp/index.html]: Would we be in the need to change the v=xy to e.g. 0.9.1 so that we force a reload for everyone?

Comment (fbonzon) [inline web/src/main/webapp/index.html]: Done.

Yes, theoretically, this is required every time main.js is updated.

**Comment by fbonzon** [comment]
Inline comments addressed.

**Comment by karussell** [comment]
Thanks!

**Pull request merged**